### PR TITLE
Split homepage VR tests into old and new

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Related to #1433 by @panchovm

## Description

This PR was created to make the #2075 easier to review by removing the changes to the homepage snapshots that are unrelated to the new layout changes.

## Testing Instructions

~~This PR is failing an e2e test in the CI because on smaller screens on the new homepage, it expects to see the search type popover, but there's still a row of button from the old design.
I think it's better not to add any changes, because we'd need to revert those changes in #2075 later. It's easier to just merge this PR as part of #2075.~~ - I've commented out the new tests, so now the tests here pass.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
The spec changes make sense, thanks for splitting it out into a more easily reviewable PR.

File: test/playwright/utils/navigation.ts [hunk 1]
```diff
@@ -319,9 +319,22 @@ export const dismissTranslationBanner = async (page: Page) => {
 export const selectHomepageSearchType = async (
   page: Page,
   searchType: SupportedSearchType,
-  dir: LanguageDirection = "ltr"
+  dir: LanguageDirection = "ltr",
+  headerMode: HeaderMode = NEW_HEADER
 ) => {
   const pageWidth = page.viewportSize()?.width
+  // TODO: replace when the search types can only be selected using the popover/modal
+  console.log("Selecting homepage search type in header mode", headerMode)
```
Comment (zackkrida) [inline test/playwright/utils/navigation.ts:327]:
```suggestion
  // console.log("Selecting homepage search type in header mode", headerMode)
```

Any reason to keep this?

**Review by zackkrida — APPROVED** [review]
Nice! LGTM.

File: test/playwright/utils/navigation.ts (hunk 1 above)
Comment (obulat) [reply test/playwright/utils/navigation.ts:327]: It's removed back in the further PRs :)

**Pull request merged**